pkg/arm/v4/data/master-startup.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash -ex
 
 if ! grep /var/lib/docker /etc/fstab; then
+{{- if ne .Config.PluginVersion "v4.2" }}
+  systemctl stop docker-cleanup.timer
```

Comment:
this wouldn't interrupt the docker-cleanup process if it's currently running, would it?